platform/persistence/platform-persistence-core-impl/src/main/java/org/codice/ddf/persistence/internal/PersistentStoreImpl.java

```diff
@@ -289,26 +264,25 @@ public int delete(String type, String cql) throws PersistenceException {
         return idsToDelete.size();
     }
 
-    private SolrClient getSolrCore(String storeName) {
-        if (coreSolrClients.containsKey(storeName)) {
-            LOGGER.debug("Returning core {} from map of coreSolrClients", storeName);
-            return coreSolrClients.get(storeName);
+    private SolrClient getSolrClient(String storeName) {
```

Comment:
@lessarderic Why would we need another `ConcurrentHashMap<SolrClient>` map?  Instead of having to manage two maps,  I think it would be ok to just have `ConcurrentHashMap<String, Future<SolrClient>>` and get the client each time from the future.